lib/web_ui/lib/src/engine/window.dart

```diff
@@ -171,7 +171,7 @@ class EngineFlutterWindow extends ui.SingletonFlutterWindow {
           assert(arguments != null);
           browserHistory.setRouteName(
             arguments!.tryString('location'),
-            state: arguments.tryString('state'),
+            state: arguments['state'],
```

Comment:
@yjbanov should we add a `tryObject()`?

Reply:
what is the point of tryObject though? it is already an object?

Reply:
Without a type, it's `dynamic` not `Object?`. Not sure if that's an important distinction or not.

Reply:
Btw, feel free to submit the PR. If we need to change it to `Object?` we can do so later.

Reply:
I can add that if that makes sense, btw what is the motive to avoid `as String?` and use `tryString`